composer.json

```diff
@@ -21,7 +21,8 @@
         "predis/predis":           "~1.0"
     },
     "conflict": {
-        "doctrine/common": ">2.2,<2.4"
+        "doctrine/common": ">2.2,<2.4",
+        "ext-apcu":        "<4.0.8"
```

Comment:
this would forbid people to install this version if they have apcu installed, even though they want to use the RedisCache. So this has indeed an unwanted impact